go/serialization_test.go

```diff
@@ -436,6 +438,16 @@ func TestSerializeSCT(t *testing.T) {
 	}
 }
 
+func TestSerializeSCTList(t *testing.T) {
```

Comment:
This test should test more of the error cases, like list too large, SCT to large, empty list, and maybe anything that could cause `SerializedLength` or `SerializeSCT` to fail, if it's relatively straightforward to cause those to fail (i.e. doesn't require extensive mocking).